src/styles/components/_component-card.scss

```diff
@@ -77,6 +75,11 @@
         top: 0;
         left: -1rem; // 16px/16
       }
+
+      letter-spacing: 2%;
+      text-align: center;
+      display: block;
+      border-top: 0.25rem solid var(--foreground-color);
```

Comment:
These can all get deleted because they're getting applied already to the header in about lines 53-ish.

Reply:
Actually, I didn't see the letter-spacing:2% anywhere else but when I removed it I saw no difference whatsoever. So I guess it's better to not have it!

Reply:
You know, on second thought it appears that 2% came from the rebase so it should probably go back in. I just looked on main, and it has been updated to letter-spacing: 2em. So, I think I will just add it back in on my next commit and when we rebase we will just go with whatever is on main.